content/the-ember-times-issue-171.md

```diff
@@ -0,0 +1,145 @@
+---
+title: The Ember Times - Issue No. 171
+author:
+  - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+date: 2021-01-15T00:00:00.000Z
+tags:
+  - newsletter
+  - ember-js-times
```

Comment:
@amyrlam It looks like we don't need to use `ember-js-times` tag anymore. The test in CI failed with the message,

```
You have defined a post with tag "ember-js-times" but there is no tag with that id. To create this tag run 'npx ember g tag ember-js-times'
```

Let's update the template in a separate pull request, then rebase this branch?

https://raw.githubusercontent.com/ember-learn/ember-blog/master/post-templates/embertimes-template.md

Reply:
Ah, you made a [PR](https://github.com/ember-learn/ember-blog/pull/849) already. 🙂

Reply:
👍 removed it here!